.golangci.yml

```diff
@@ -1,84 +1,304 @@
-# This file contains all available configuration options
-# with their default values.
-
-# options for analysis running
 run:
-  tests: true
-  timeout: 10m
+  # Timeout for analysis, e.g. 30s, 5m.
```

Comment:
Could we base the config on the original one instead 
https://gist.githubusercontent.com/maratori/47a4d00457a92aa426dbd48a18776322/raw/309ab85402cd3e8b6e4a9a515a57c6baa8082108/.golangci.yml

and leave the header comment so we know where this comes from?

This needs updating golangci version to 1.59.1 as the config was updated for this golangci version.